AppAuth-JS/built/node_support/crypto_utils.d.ts

```diff
@@ -0,0 +1,2 @@
+import { RandomGenerator } from '../crypto_utils';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork